### DISCUSSION TITLE
Closed Source vs. Open Source

### DISCUSSION THREAD

**Discussion by alizezila** [albertkun/21S-ASIAAM-191A#111] [Past Thinking Caps]
Close source tools are harmful to society because it dampens opportunities for innovation and collaboration. Furthermore, the commodification of knowledge through patents and closed source coded technology is a process that is imbued with power and access to resources. Open source tools are more equitable and accessible which allows people to collaborate and add input about tools that may be lacking in some areas and can help address them. A pro-capitalism argument is that without patents and the commodification of knowledge, people lack incentive to create and innovate. Personally, I think that is very narrow-minded and that people are always driven to create to help others and help society. Open-source tools better allow people to do that kind of work and not be inhibited by a paywall.

**Comment by latishasumardy** [comment]
I agree with your point about closed source tools. I believe that the secrecy of it really deters people who may want to build their own software, as it discourages them from trying. Also, the secrecy of the code can be detrimental, as people who find bugs in the software have no way to fix it themselves, and they must rely on the company to fix it. I also agree that open source tools are better because it allows for a community of people who use it to grow and collaborate with each other, increasing the rate at which bugs are found and resolved. It also allows people to be more creative and create their own features and tools, which is not possible with closed source tools.
